pkg/sessionctx/variable/sysvar.go

```diff
@@ -211,7 +211,7 @@ var defaultSysVars = []*SysVar{
 		}
 		return nil
 	}},
-	{Scope: ScopeSession, Name: TiDBOptProjectionPushDown, Value: BoolToOnOff(config.GetGlobalConfig().Performance.ProjectionPushDown), Type: TypeBool, SetSession: func(s *SessionVars, val string) error {
+	{Scope: ScopeGlobal | ScopeSession, Name: TiDBOptProjectionPushDown, Value: BoolToOnOff(DefOptEnableProjectionPushDown), Type: TypeBool, SetSession: func(s *SessionVars, val string) error {
```

Comment:
Why do we need to change this? It appears that the old behavior has changed.

Reply:
To allow different init values for new deployed 8.3.0 tidb cluster and upgraded 8.3.0 tidb cluster, need to change to global session scope. Besides, it makes sense to be a global session scope system variable also.